server/portal/apps/auth/models.py

```diff
@@ -75,7 +75,9 @@ def client(self):
                      client_id=getattr(settings, 'TAPIS_CLIENT_ID'),
                      client_key=getattr(settings, 'TAPIS_CLIENT_KEY'),
                      access_token=self.access_token,
-                     refresh_token=self.refresh_token)
+                     refresh_token=self.refresh_token,
+                     resource_set='dev',
+                     download_latest_specs=True)
```

Comment:
Just bumped tapipy to 1.3.0, so we can lose `download_latest_specs=True`.

Should be able to drop `resource_set='dev'` once it gets released to prod